src/i18n/es.json

```diff
@@ -4,10 +4,55 @@
   "translation": {
     "reaction-accounts": {
       "admin": {
+        "accounts": {
+          "tabs": {
+            "staff": "Empleados",
+            "customers": "Clientes",
+            "invites": "Invitaciones"
+          },
+          "accountsLabel": "Cuentas",
+          "createGroup": "Crea un grupo",
+          "inviteStaffMember": "Invitar empleado"
+        },
+        "accountsTable": {
+          "header": {
+            "email": "Dirección de correo electrónico",
+            "name": "Nombre"
+          },
+          "bulkActions": {
+            "addRemoveGroupsFromAccount": "Agregar/Quitar grupos de la cuenta"
+          }
+        },
+        "invitationsTable": {
+          "header": {
+            "email": "Dirección de correo electrónico",
+            "invitedBy": "Invitado por",
+            "shop": "Tienda",
+            "groups": "Grupos"
+          }
+        },
+        "groupCards": {
+          "createOrUpdateGroupDialog": {
+            "title": "Crear/Actualizar grupo",
+            "name": "Nombre",
+            "slug": "Slug",
```

Comment:
I know there is no direct translation for this, but I've been using "Identificador" for other plugins. What do you think about using that instead?

Reply:
Perfect, that was the word I was looking for